tests/client/test_client.py

```diff
@@ -0,0 +1,180 @@
+import pandas as pd
+import pytest
+from django.core.cache import cache
+from django.test import LiveServerTestCase, TestCase
+
+from hawc_client import BaseClient, HawcClient
+
+
+"""
```

Comment:
hey sorry, one super-nit request. can this comment please be moved to be the class-level docstring, and can you make sure the line-width is <100, eg., 

```python
@pytest.mark.usefixtures("set_db_keys")
class TestClient(LiveServerTestCase, TestCase):
    """
    We use a single class that inherits from both LiveServerTestCase and TestCase 
    in order to supersede properties of LiveServerTestCase that cause the database to be flushed 
    after every test, while still being able to utilize a live server for HTTP requests.

    Further reading: https://code.djangoproject.com/ticket/23640#comment:3
    """
```

After that feel free to merge!